README.md

```diff
@@ -35,12 +35,17 @@ sure the [prerequisites](#Prerequisites) are installed first.
 3. Install the
    [KnativeServing custom resource](#the-knativeserving-custom-resource)
 
+```sh
+kubectl create namespace knative-serving
```

Comment:
How do you feel about replacing this with the following inside the `cat <<-EOF`:
```
apiVersion: v1
kind: Namespace
metadata:
 name: knative-serving
---
```

Reply:
Sounds good. done